internal/core/metadata/init.go

```diff
@@ -37,29 +42,33 @@ import (
 
 // Bootstrap contains references to dependencies required by the BootstrapHandler.
 type Bootstrap struct {
-	router *mux.Router
+	muxRouter            *mux.Router
+	inDebugMode          bool
+	inAcceptanceTestMode bool
 }
 
 // NewBootstrap is a factory method that returns an initialized Bootstrap receiver struct.
-func NewBootstrap(router *mux.Router) *Bootstrap {
+func NewBootstrap(muxRouter *mux.Router, inDebugMode, inAcceptanceTestMode bool) *Bootstrap {
 	return &Bootstrap{
-		router: router,
+		muxRouter:            muxRouter,
+		inDebugMode:          inDebugMode,
+		inAcceptanceTestMode: inAcceptanceTestMode,
 	}
 }
 
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the metadata service.
 func (b *Bootstrap) BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ startup.Timer, dic *di.Container) bool {
-	loadRestRoutes(b.router, dic)
-
 	configuration := container.ConfigurationFrom(dic.Get)
-
-	// initialize clients required by service.
+	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
 	registryClient := bootstrapContainer.RegistryFrom(dic.Get)
 
+	loadRestRoutes(b.muxRouter, dic)
```

Comment:
in core command you changed this function to loadV1Routes(...)
for consistency, it would be better to change this for core data and metadata, etc as well